sroka/api/google_ad_manager/gam_api.py

```diff
@@ -155,3 +155,65 @@ def get_users_from_admanager(query, dimensions, network_code=None):
     except errors.AdManagerReportError as e:
         print('Failed to generate user list. Error was: {}'.format(e))
         return
+
+def get_companies_from_admanager(query, dimensions, network_code=None):
+
+    company_df = pd.DataFrame()
```

Comment:
Here we have code duplication with earlier functions. We should remove the redundancies. However, we can move now with this solution, but let's do refactor prior to the next release. Do you want to do it, or should I put it in my TODO list?

Reply:
Done